scss/style.scss

```diff
@@ -1,23 +1,138 @@
+body {
```

Comment:
I would suggest improving the calculator color palette because currently, it looks little bit scary :grinning: You can use already existing tools for it (e.g. [Coolors](https://coolors.co/)).